src/app/phase-team-response/issues-pending/issues-pending.component.ts

```diff
@@ -58,10 +58,9 @@ export class IssuesPendingComponent implements OnInit, OnChanges {
   }
 
   ngOnInit() {
-    this.filter = (issue: Issue) => {
-      return (!this.issueService.hasTeamResponse(issue.id) || (!issue.status || issue.status === 'Incomplete')) &&
-        !issue.duplicateOf;
-    };
+    const isNotDuplicate = (issue: Issue) => !issue.duplicateOf;
+    const doesNotHaveResponse = (issue: Issue) => (!this.issueService.hasTeamResponse(issue.id) || (!issue.status || issue.status === 'Incomplete'));
```

Comment:
For the `issue.status === 'Incomplete'` perhaps we could use `issue.status === STATUS.Incomplete` instead since we have a pre-defined enum 👍